### PR TITLE
Adding new testimonial component

### PR DESCRIPTION
**Pull request by SanjeebNath382**
Hey thanks for great youtube video, i am adding a new testimonial component

### CODE REVIEW

File: components/testimonial-sanjeeb.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+      </svg>
+      <p class="leading-relaxed text-lg">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas lorem ex, pretium non sodales at, feugiat eget magna. Morbi quis rhoncus leo, in semper metus. Curabitur vulputate tempor dui. Praesent fringilla volutpat massa vel iaculis. Sed dui nibh, dapibus et sodales pellentesque, posuere a leo. Nunc euismod suscipit est, vitae aliquet augue pellentesque ut. In convallis dolor eget dui lobortis varius. Nullam tempor enim quis blandit porttitor. Mauris sit amet ullamcorper purus. In malesuada odio nec quam vehicula molestie. Vivamus vitae sollicitudin justo. Donec pulvinar nec turpis quis commodo. Morbi cursus risus a rutrum laoreet. Aenean erat erat, tincidunt a arcu tincidunt, porttitor congue diam. Mauris ac tincidunt dui. Cras sed vehicula augue. Aenean porta malesuada diam, vel facilisis sem auctor quis. Suspendisse lectus magna, volutpat id velit sit amet, sodales mollis metus. Ut elementum porta ipsum, condimentum ullamcorper nisi venenatis in. </p>
```
Comment (thinkverse) [inline components/testimonial-sanjeeb.html]: A more realistic testimonial than a lorem ipsum dump would demonstrate the component better. 🙂

**Review by thinkverse — CHANGES_REQUESTED** [review]
A better name for this file would be `testimonial-single.html` since it's a single testimonial. That would also follow the [naming convention] for components mentioned in the CONTRIBUTING file.

[naming convention]: https://github.com/thedevdojo/tails/blob/HEAD/CONTRIBUTING.md#naming-conventions

**Comment by tnylea** [comment]
Thanks man 🙌 Will get this one merged in, in the morning 😊

**Comment by SanjeebNath382** [comment]
thanks man!

**Review by thinkverse — APPROVED** [review]

**Pull request merged**